packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -327,55 +331,44 @@ describe('account', () => {
-                            },
-                            supply: expect.any(String),
+                        accountSpace: 80n,
+                        accountType: 'initialized',
+                        address: 'AiZExP8mK4RxDozh4r57knvqSZgkz86HrzPAMx61XMqU',
+                        authority: {
+                            address: '3xxDCjN8s6MgNHwdRExRLa6gHmmRTWPnUdzkbKfEgNAe',
                         },
-                        meta: {
-                            program: 'spl-token',
-                            space: 82n,
-                            type: 'mint',
+                        blockhash: expect.any(String),
+                        feeCalculator: {
+                            lamportsPerSignature: expect.any(String),
                         },
+                        lamports: expect.any(BigInt),
+                        programName: 'nonce',
                     },
                 },
             });
         });
-        it('can get a token account', async () => {
+        it('can get an address lookup table account', async () => {
             expect.assertions(1);
-            // See scripts/fixtures/spl-token-token-account.json
+            // See scripts/fixtures/address-lookup-table-account.json
             const variableValues = {
-                address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
+                address: '2JPQuT3dHtPjrdcbUQyrrT4XYRYaWpWfmAJ54SUapg6n',
             };
             const source = /* GraphQL */ `
                 query testQuery($address: String!) {
                     account(address: $address) {
-                        ... on TokenAccount {
-                            data {
-                                isNative
-                                mint {
-                                    address
-                                }
-                                owner {
-                                    address
-                                }
-                                state
-                                tokenAmount {
-                                    amount
-                                    decimals
-                                    uiAmount
-                                    uiAmountString
-                                }
-                            }
-                            meta {
-                                program
-                                space
-                                type
+                        address
+                        lamports
+                        ... on LookupTableAccount {
+                            addresses
+                            authority {
+                                address
                             }
+                            deactivationSlot
+                            lastExtendedSlot
+                            lastExtendedSlotStartIndex
+                            programName
+                            accountSpace
```

Comment:
This stuff should just be on the base type, right?

Also, why `account.accountSpace`. Were you worried of a collision with the name `account.space`?

Reply:
> This stuff should just be on the base type, right?

Was it just for the older versions that `space` was only available with `jsonParsed`? Because I didn't see this in the default account type from the payloads I was checking out.

> Also, why `account.accountSpace`. Were you worried of a collision with the name `account.space`?

Yeah, wanted to make sure, although there's no other accounts in our lib that have `space`, if someone rolled pluggable parsing they didn't have issues. `space` is a pretty generic field name.

Reply:
nvm, I'm going to cross this bridge if/when we get there. Reverted back to `space`.